app/models/volunteer_op.rb

```diff
@@ -34,4 +42,16 @@ def self.contains_description?(text)
   def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
+  
+  def address_complete?
+    address.present? && postcode.present?
+  end 
+  
+  private
+  
+  def clear_lat_lng
+    return if address_complete?
+    self.longitude = nil
+    self.latitude = nil
+  end
```

Comment:
@Maroo-b maybe, although by inserting the little conditional on the clear_lat_lng you suggested in one of your other pull requests, we seem to have things under control - in this PR at least.  I'm not clear if that's resolved the problem  - I guess the next steps are to merge this - and then start pushing it towards production; and see if we can have a cucumber test that catches the problem (if it still remains ...)